.github/workflows/ubuntu.yml

```diff
@@ -132,13 +132,12 @@ jobs:
            cd _build
            ctest -C Release --output-on-failure
 
-      # simtest
       - name: Read simtest version
-        id: simtest-version
-        uses: notiz-dev/github-action-json-property@release
-        with:
-          path: 'simtest.json'
-          prop_path: 'version'
+        run: |
+          echo 'simtest<<EOF' >> $GITHUB_ENV
+          cat ./simtest.json >> $GITHUB_ENV
+          echo 'EOF' >> $GITHUB_ENV
+          echo "SIMTEST=${{ fromJson(env.SIMTEST_JSON).version }}" >> $GITHUB_ENV
```

Comment:
The step that creates or updates the environment variable does not have access to the new value, but all subsequent steps in a job will have access

https://stackoverflow.com/a/75433159